src/game/title_screen.c

```diff
@@ -1729,44 +1731,53 @@ static void Task_LensFlareAnim(void)
     };
 }
 
-#define TinyChaoGardenConfig ((u32 *)(EWRAM_START + 0x8))
-
+#include "../chao_garden/include/language.h"
+#include "../chao_garden/include/program_params.h"
```

Comment:
Not sure I like the relative includes. We can add chao_garden include to the `-I` path and then name the headers something with `chao_garden` in the name, or we can have a folder called `chao_garden` in there?

Reply:
Also, why are these not at the top?

Reply:
I'm also for just not passing headers to SA2 since they likely just hard coded these anyway :/